eng/native/naming.props

```diff
@@ -2,9 +2,36 @@
   <PropertyGroup>
     <StaticLibPrefix>lib</StaticLibPrefix>
     <ExeSuffix Condition="'$(HostOS)' == 'windows'">.exe</ExeSuffix>
+    <Blah>$(HostOS)</Blah>
   </PropertyGroup>
 
   <!-- Add path globs specific to native binaries to exclude unnecessary files from packages. -->
+  <Choose>
```

Comment:
Doesn't this logic already exist?

@jkoritzinsky ?

Reply:
I don't believe it exists elsewhere. I put this here because I found host / target confusion when crossgen2 was cross building android from a mac.